airbyte-cdk/python/airbyte_cdk/sources/streams/http/error_handlers/default_backoff_strategy.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+
+from typing import Optional, Union
+
+import requests
+
+from .backoff_strategy import BackoffStrategy
+
+
+class DefaultBackoffStrategy(BackoffStrategy):
+    def __init__(
+        self,
+        max_retries: int = 5,
+        max_time: int = 60 * 10,
```

Comment:
It feels like it would be more intuitive to use a `timedelta`. Using a `int`, it is very hard to understand the units for that and it feels error prone

Reply:
I've updated type to `timedelta` and then in the `__init__` function added a conversion to seconds (via `.total_seconds()`).